internal/provider/resource_cloudflare_account_member.go

```diff
@@ -101,7 +101,8 @@ func resourceCloudflareAccountMemberCreate(ctx context.Context, d *schema.Resour
 		accountID = client.AccountID
 	}
 
-	r, err := client.CreateAccountMember(ctx, accountID, memberEmailAddress, accountMemberRoleIDs)
+	status := d.Get("status").(string)
```

Comment:
you'll need to add this field to the schema - https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/schema_cloudflare_account_member.go

i'd also recommend defaulting to `""` and having a conditional in here that if the status is not empty (the default), use the new method with the status.

Reply:
Ok, thanks, I'll add the schema field. doesn't `d.Get("status").(string)` default to "" for me?

Reply:
> The function CreateAccountMember in cloudflare-go simply calls CreateAccountMemberwithStatus using "" as the status. [cloudflare/cloudflare-go@master/account_members.go#L114](https://github.com/cloudflare/cloudflare-go/blob/master/account_members.go?rgh-link-date=2022-09-22T08%3A39%3A58Z#L114) , so as long as d.Get("status").(string) results in "", then I don't think a conditional is required here. What do you think?

yep, that works for me! thanks for digging into it.